### PR TITLE
ide+vscode: Show full function signature

### PR DESCRIPTION
**Pull request by Lenart12**
Before it just displayed `function name()`, now it displays the whole signature.

### CODE REVIEW

File: editors/vscode/server/src/server.ts [hunk 1]
```diff
@@ -138,8 +139,15 @@ connection.onHover(async (request) => {
 			console.log("hovering");
 			console.log(obj);
 
+			// FIXME: Figure out how to import `vscode` package in server.ts without
+			// getting runtime import errors to remove this deprication warning.
+			const contents : MarkedString=  {
+				value: `${obj.hover}`,
```
Comment (u9g) [inline editors/vscode/server/src/server.ts:145]:
```suggestion
				value: obj.hover,
```

File: editors/vscode/server/src/server.ts [hunk 2]
```diff
@@ -138,8 +139,15 @@ connection.onHover(async (request) => {
 			console.log("hovering");
 			console.log(obj);
 
+			// FIXME: Figure out how to import `vscode` package in server.ts without
+			// getting runtime import errors to remove this deprication warning.
+			const contents : MarkedString=  {
```
Comment (u9g) [inline editors/vscode/server/src/server.ts:144]:
```suggestion
			const contents: MarkedString =  {
```

**Review by u9g — CHANGES_REQUESTED** [review]

**Comment by Lenart12** [comment]
Thanks @u9g I fixed it up 👍

**Comment by jntrnr** [comment]
thanks!

**Pull request merged**